go.mod

```diff
@@ -73,13 +73,13 @@ require (
 	go.uber.org/atomic v1.9.0 // indirect
 	go.uber.org/multierr v1.7.0 // indirect
 	go.uber.org/zap v1.24.0 // indirect
-	golang.org/x/mod v0.10.0 // indirect
-	golang.org/x/net v0.10.0 // indirect
+	golang.org/x/mod v0.12.0 // indirect
```

Comment:
Looks like there is a separate PR for this upversion, which I think is a more appropriate place for it:
https://github.com/openshift-kni/cluster-group-upgrades-operator/pull/660

Maybe we can get that PR merged first, then rebase against it? I don't see PRs from the dependbot for these other modules. These upversions seem unrelated to the code changes here, so it would be nice to have these done separately

Reply:
It's merged now.

Reply:
Oh, right, the other PR is for x/sys

Reply:
Should I create a separate PR for `x/net`?

Reply:
That's just my two cents. I prefer to see upversioning done in a separate PR, primarily to isolate the actual code changes, whether for future cherry-pick or readability, or even for association when the upversion is unrelated to the PR. But it's up to @jc-rh . Everything else looks fine to me.

Reply:
These are indirect dependencies. Sharat mentioned it was updated automatically due to `unix.Mkdev` he used in this fix. I assume that means it's valid to include the upversion in this PR?

Reply:
I'm guessing dependbot hasn't updated these already because of the configuration, that the upversion hasn't been deemed critical and so isn't done automatically?

Reply:
Not sure why. Maybe it was blocked by other dependbot PRs? We have a few pending ones that require manual updates in our code.